src/algokit/cli/generate.py

```diff
@@ -3,12 +3,72 @@
 
 import click
 
+from algokit.core.generate import load_generators, run_generator
 from algokit.core.typed_client_generation import ClientGenerator
 
 logger = logging.getLogger(__name__)
 
 
-@click.group("generate")
+def _load_custom_generate_commands(project_dir: Path) -> dict[str, click.Command]:
+    """
+    Load custom generate commands from .algokit.toml file.
+    :param project_dir: Project directory path.
+    :return: Custom generate commands.
+    """
+
+    generators = load_generators(project_dir)
+    commands_table: dict[str, click.Command] = {}
+
+    for generator in generators:
+
+        @click.command(
+            name=generator.name, help=generator.description or "Generator command description is not supplied."
+        )
+        @click.option(
+            "answers",
+            "--answer",
+            "-a",
+            multiple=True,
+            help="Answers key/value pairs to pass to the template.",
+            nargs=2,
+            default=[],
+            metavar="<key> <value>",
+        )
+        @click.option(
+            "path",
+            "--path",
+            "-p",
+            help=f"Path to {generator.name} generator. (Default: {generator.description})",
```

Comment:
should the default refer to {generator.path} ?

Reply:
@daniel-makerx just pushed a fix, thanks for catching this typo